src/features/currentTodo.ts

```diff
@@ -1,10 +1,19 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSelector, createSlice } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
+import { RootState } from '../app/store';
 
 const initialState = null as Todo | null;
 
 export const currentTodoSlice = createSlice({
   name: 'currentTodo',
   initialState,
-  reducers: {},
+  reducers: { set: (_, { payload }) => payload },
```

Comment:
The `set` reducer directly assigns the `payload` to the state. Ensure that the `payload` is always of type `Todo` or `null` to maintain type safety. Consider adding type checks or validation if necessary.